.github/workflows/wheel.yml

```diff
@@ -6,6 +6,9 @@ on: [push, pull_request]
 jobs:
```

Comment:
This changes should be in a separate pull request.

Reply:
https://github.com/SymbiFlow/fasm/pull/33